osquery/CMakeLists.txt

```diff
@@ -69,10 +69,11 @@ endif()
 
 if(APPLE OR LINUX)
   ADD_OSQUERY_LINK_CORE("libdl")
-  ADD_OSQUERY_LINK_CORE("boost_system-mt")
-  ADD_OSQUERY_LINK_CORE("boost_filesystem-mt")
+  ADD_OSQUERY_LINK_CORE("boost_system")
```

Comment:
If you rename these it will break Apple and all other Linux builds. Our dependency setup scripts should make these library available.